components/tools/OmeroWeb/omeroweb/api/decorators.py

```diff
@@ -75,7 +76,7 @@ def handle_error(self, ex, trace):
         # But we try to handle all 'expected' errors appropriately
         # TODO: handle omero.ConcurrencyException
```

Comment:
Yes. Hard to see that we need to support ConcurrencyException since we are doing very simple get/set queries now and no stateful services.